static/js/components/dashboard/CourseAction.js

```diff
@@ -57,34 +96,30 @@ export default class CourseAction extends React.Component {
       action = <i className="material-icons">done</i>;
       break;
     case STATUS_CAN_UPGRADE: {
-      action = this.makeEnrollButton("Upgrade", firstRun, false);
+      let formattedUpgradeDate = moment(firstRun.upgrade_deadline).format(DASHBOARD_FORMAT);
```

Comment:
changed... but do you have any idea why that is?? seems so arbitrary. we're literally just changing the key from `upgrade_deadline` to `course_upgrade_deadline` before returning the results.